go/general: comments for disapprove (#1223)
exercism/website-copy: automated-comments/go/general/golint_not_satisfied.md

There's a great tool called [`golint`](https://github.com/golang/lint) which will examine the code for common problems and style issues.

> Try running `golint` on the solution, e.g.: `golint two_fer.go`; it will make some useful suggestions.

Reviewer: ```suggestion
Run `golint` on the solution, e.g.: `golint two_fer.go`; it will make some useful suggestions.
```

The problem with `try` is that, if it fails, a student might leave it. With an imperative it's more likely that they'll complain if it doesn't work. (I have no proof for this 😅, only anecdotal evidence).

> It's a good idea to always check ones code with `golint`.

Reviewer 2: ```suggestion
It's a good idea to always check one's code with `golint`.
```

Or perhaps just:

```suggestion
It's a good idea to always check the code with `golint`.
```

When writing Go software for production, many build pipelines will automatically fail if the code doesn't pass `golint`.

If you are keen on getting much more feedback from code linters there is the [golangci-lint](https://github.com/golangci/golangci-lint) project
allowing you to run a lot of linters with one command.

> You can create a config file for your project to adjust the behaviour to your needs.

Reviewer: ```suggestion
A project config file can adjust the behaviour to a project's needs.
```
